src/CMakeLists.txt

```diff
@@ -1038,6 +1038,12 @@ if(WITH_RADOSGW)
   add_dependencies(vstart radosgw radosgw-admin)
 endif(WITH_RADOSGW)
 
+if(WITH_LTTNG)
+  add_dependencies(vstart rados_tp)
+  add_dependencies(vstart os_tp)
+  add_dependencies(vstart osd_tp)
+  add_dependencies(vstart rbd_tp)
+endif(WITH_LTTNG)
```

Comment:
Good point. AFAIK, I can't add a dependency directly on vstart in tracing/CMakeLists.txt due to the cmake structure, so I used a dependency indirection using tracepoint_libraries.